app/Http/Controllers/Api/Manager/ChapterController.php

```diff
@@ -190,24 +190,16 @@ public function delete(DeleteRequest $request): JsonResponse
      */
     public function bulkDelete(BulkDeleteRequest $request, BulkDeleteChapterService $bulkDeleteChapterService): JsonResponse
     {
-        // ログイン中の講師IDを取得
-        $managerId = Auth::guard('instructor')->user()->id;
-
-        // マネージャーが管理する講師を取得
-        $manager = Instructor::with('managings')->find($managerId);
-        $instructorIds = $manager->managings->pluck('id')->toArray();
-        $instructorIds[] = $manager->id;
-
         $chapterIds = $request->input('chapters', []);
         $courseId = $request->input('course_id');
 
         try {
             $chapters = Chapter::with(['course', 'lessons'])->whereIn('id', $chapterIds)->get();
-            $chapters->each(function (Chapter $chapter) use ($instructorIds, $courseId) {
-                if (! in_array($chapter->course->instructor_id, $instructorIds, true)) {
-                    // 自分、または配下の講師の講座のチャプターでなければエラー応答
-                    throw new AuthorizationException('Forbidden, invalid instructor_id.');
-                }
+            $chapters->each(function (Chapter $chapter) use ($courseId) {
+
+                // 認可処理 policy使用
+                $this->authorizeResource('bulkDelete', [Chapter::class, $chapter]);
+
```

Comment:
こちらも段階的に認可処理を集約していくので、次のようなコードになる想定でした！

```suggestion
            // 認可処理 policy使用
            $this->authorize('bulkDelete', [Chapter::class, $chapters]);
            
            $chapters->each(function (Chapter $chapter) use ($courseId) {

```